src/main/java/dk/xam/jbang/Main.java

```diff
@@ -260,6 +260,42 @@ public Integer call() throws IOException {
 		}
 	}
 
+	static boolean checkVersions() {
+		String version = System.getProperty("java.version");
```

Comment:
look for supportsModules() - here we are calling out and check if java supports modules by looking at -version string - maybe we can reuse/share that logic so we avoid calling out twice ?

Reply:
I did not understand this part a bit.
Should i look for modules that let us retrieve version of java being used?

Reply:
I mean, that in `supportModules()` there is already a call out to `javac` - optimally we would only want to call out to get the java version *once* at most.

Reply:
So to find the version for both `javac` and `java` I should use `supportModules()` method?
Also there is no call for `javac` but for `java` there is.

Reply:
yes sorry - I meant for `java`